lib/std/zig/system/linux.zig

```diff
@@ -163,61 +163,113 @@ const ArmCpuinfoImpl = struct {
+            E{ .part = 0xc18, .m32 = &A32.cortex_r8, .m64 = null },
+            E{ .part = 0xc20, .m32 = &A32.cortex_m0, .m64 = null },
+            E{ .part = 0xc21, .m32 = &A32.cortex_m1, .m64 = null },
+            E{ .part = 0xc23, .m32 = &A32.cortex_m3, .m64 = null },
+            E{ .part = 0xc24, .m32 = &A32.cortex_m4, .m64 = null },
+            E{ .part = 0xc27, .m32 = &A32.cortex_m7, .m64 = null },
+            E{ .part = 0xc60, .m32 = &A32.cortex_m0plus, .m64 = null },
+            E{ .part = 0xd01, .m32 = &A32.cortex_a32, .m64 = null },
+            E{ .part = 0xd03, .m32 = &A32.cortex_a53, .m64 = &A64.cortex_a53 },
+            E{ .part = 0xd04, .m32 = &A32.cortex_a35, .m64 = &A64.cortex_a35 },
+            E{ .part = 0xd05, .m32 = &A32.cortex_a55, .m64 = &A64.cortex_a55 },
+            E{ .part = 0xd07, .m32 = &A32.cortex_a57, .m64 = &A64.cortex_a57 },
+            E{ .part = 0xd08, .m32 = &A32.cortex_a72, .m64 = &A64.cortex_a72 },
+            E{ .part = 0xd09, .m32 = &A32.cortex_a73, .m64 = &A64.cortex_a73 },
+            E{ .part = 0xd0a, .m32 = &A32.cortex_a75, .m64 = &A64.cortex_a75 },
+            E{ .part = 0xd0b, .m32 = &A32.cortex_a76, .m64 = &A64.cortex_a76 },
+            E{ .part = 0xd0c, .m32 = &A32.neoverse_n1, .m64 = null },
+            E{ .part = 0xd0d, .m32 = &A32.cortex_a77, .m64 = &A64.cortex_a77 },
+            E{ .part = 0xd13, .m32 = &A32.cortex_r52, .m64 = null },
+            E{ .part = 0xd20, .m32 = &A32.cortex_m23, .m64 = null },
+            E{ .part = 0xd21, .m32 = &A32.cortex_m33, .m64 = null },
+            E{ .part = 0xd41, .m32 = &A32.cortex_a78, .m64 = &A64.cortex_a78 },
+            E{ .part = 0xd4b, .m32 = &A32.cortex_a78c, .m64 = &A64.cortex_a78c },
+            E{ .part = 0xd44, .m32 = &A32.cortex_x1, .m64 = &A64.cortex_x1 },
+            E{ .part = 0xd02, .m64 = &A64.cortex_a34 },
+            E{ .part = 0xd06, .m64 = &A64.cortex_a65 },
+            E{ .part = 0xd43, .m64 = &A64.cortex_a65ae },
+        };
+        // implementer = 0x42
+        const Broadcom = [_]E{
+            E{ .part = 0x516, .m64 = &A64.thunderx2t99 },
+        };
+        // implementer = 0x43
+        const Cavium = [_]E{
+            E{ .part = 0x0a0, .m64 = &A64.thunderx },
+            E{ .part = 0x0a2, .m64 = &A64.thunderxt81 },
+            E{ .part = 0x0a3, .m64 = &A64.thunderxt83 },
+            E{ .part = 0x0a1, .m64 = &A64.thunderxt88 },
+            E{ .part = 0x0af, .m64 = &A64.thunderx2t99 },
+        };
+        // implementer = 0x46
+        const Fujitsu = [_]E{
+            E{ .part = 0x001, .m64 = &A64.a64fx },
+        };
+        // implementer = 0x48
+        const HiSilicon = [_]E{
+            E{ .part = 0xd01, .m64 = &A64.tsv110 },
+        };
+        // implementer = 0x4e
+        const Nvidia = [_]E{
+            E{ .part = 0x004, .m64 = &A64.carmel },
+        };
+        // implementer = 0x51
+        const Qualcomm = [_]E{
+            E{ .part = 0x06f, .m32 = &A32.krait },
```

Comment:
Don't you have to add idiva and idivt to its features?
cf https://android.googlesource.com/platform/external/cpu_features/+/15181ac63e8f985f645d2445fff64948521d5496/src/cpuinfo_arm.c#139

Reply:
That has nothing to do with what we're doing here, the feature set in `lib/std/target/arm.zig` is correct.